conda/base/context.py

```diff
@@ -2202,77 +2201,43 @@ def _first_writable_envs_dir():
     raise NoWritableEnvsDirError(context.envs_dirs)
 
 
+@deprecated(
+    "25.3",
+    "25.9",
```

Comment:
Don't want to skip the pending deprecation window:

```suggestion
    "25.9",
    "26.3",
```